lib/gc/Analysis/MatmulConfigAnalysis.cpp

```diff
@@ -0,0 +1,438 @@
+//===-- MatmulConfigAnalysis.cpp - DESC -------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "gc/Analysis/MatmulConfigAnalysis.h"
+#include <limits>
+#include <llvm/Support/Debug.h>
+
+namespace mlir {
+namespace gc {
+
+#define DEBUG_TYPE "matmul-config-analysis"
+
+llvm::raw_ostream &operator<<(llvm::raw_ostream &ss,
+                              const MatmulConfig &config) {
+
+  ss << "MBlock: " << config.MBlock << ", NBlock: " << config.NBlock
```

Comment:
Is it better print the configs in the order of loop nest?

Reply:
Thanks for the advice! The order has been changed according to the loop order